packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,54 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+    . $toolsPath\sfta.ps1
+    . $toolsPath\helpers.ps1
+
+    $downloadDir = Get-PackageCacheLocation
+    $installer          = 'googlechromeinstaller.msi'
+    $packageArgs        = @{
+        packageName     = $env:ChocolateyPackageName
+        file            = Join-Path $downloadDir $installer
+        url             = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise.msi'
+        url64bit        = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
+        fileType        = 'MSI'
+        silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+        validExitCodes  = @(0)
+    }
+
+    # Download the installer
+    $packageArgs['file'] = Get-ChocolateyWebFile @packageArgs
+
+    $sigValid = (Get-AuthenticodeSignature -FilePath $packageArgs['file']).Status -eq 'Valid'
+
+    if ($sigValid) {
+        $toolName = 'Google Chrome'
+        $category = 'Utilities'
+
+        VM-Write-Log "INFO" "Signature valid ... installing as $category" # for lint check
+        Install-ChocolateyInstallPackage @packageArgs
+
+        VM-Write-Log "INFO" "Setup associations"
+        Set-PTA ChromeHTML http
+        Set-PTA ChromeHTML https
+        Set-FTA ChromeHTML .htm
+        Set-FTA ChromeHTML .html
+
+        VM-Write-Log "INFO" "Disable update settings"
+        Disable-Chrome-Updates
+
+        # Delete Desktop shortcut
+        $desktopShortcut = Join-Path ${Env:Public} "Desktop\$toolName.lnk"
```

Comment:
I think the correct path is
```suggestion
        $desktopShortcut = Join-Path ${Env:UserProfile} "Desktop\$toolName.lnk"
```

Reply:
`$Env:Public` is correct one, the installer created `.lnk` is in the public folder, however I just noticed when open the Chrome for the first time, it will create another `.lnk` which is in `$Env:UserProfile` 😂

Reply:
I think this would be a good opportunity to add a `VM-Delete-Desktop-Shortcut` function that just checks both directories. Maybe not required for this PR but that function has been on my list for a while

Reply:
I just tested again, the installer created `.lnk` file is in the public folder, after install `cyberchef.vm` and open the `.html` file, a new Chrome `.lnk` shortcut file appeared in `$Env:UserProfile`. This looks infeasible for post install deletion.

Reply:
We could open chrome in the installer, close it, and then delete the lnk.